.github/workflows/msys2.yml

```diff
@@ -0,0 +1,50 @@
+name: MSYS2 Tests
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+jobs:
+  test:
+    name: Test with MSYS2 ${{ matrix.sys }}
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        include:
+          # - { sys: msys }, setup fails
+          # - { sys: mingw32 }, impossible asm constraints when building base64
+          - { sys: mingw64 }
+          - { sys: ucrt64  }
+          - { sys: clang32 }
```

Comment:
Actually, We are in process in dropping 32-bit so no need to test mingw32 & clang32. better to remove them